docs/features/jsonSchema.md

```diff
@@ -0,0 +1,56 @@
+# JSON Schema 
+
```

Comment:
I would suggest adding a disclaimer right here. 

```suggestion

| WARNING: This page describes a beta feature which is currently under development. The implementation may change in an incompatible way. Use at your own risk |
| --- |

Jenkins Configuration as Code plugin sometimes requires users to define pretty complex YAML configuration files.
At the same time, mistakes in the configuration may lead to the instance instability.
In order to help maintainers, the plugin can generate an instance-specific JSON schema which can be used for validation of YAML files.

## Current state

Currently there are 2 types of JSON schema being generated by the plugin.
There is a legacy schema generation logic which has been available since JCasC 1.0, but this feature generates an invalid JSON schema which cannot be used as is without manual patches.
In order to solve the issue, in the [JCasC Developer Tools project](https://jenkins.io/projects/jcasc/dev-tools/) we are working on a new Schema version which is currently available as a **beta feature**.
The implementation may change in an incompatible way before the final release.

The documentation below describes the new implementation.

```